### ISSUE TITLE
[Feature]: Option to remove back button from nav bar

### ISSUE THREAD

**Issue by NikunjKhangwal** [fast4x/RiMusic#1493]
### Description

In-app back button isn't needed that much because we can use device back button

### Suggested Solution

Option to remove it so nav bar will look even less cluttered

![IMG_20240413_090315](https://github.com/fast4x/RiMusic/assets/99175083/5f4beb66-24a8-4a4b-93d8-a5a9a95d2dec)

**Comment by sayyid5416** [comment]
Yeah, this button is not really necessary. Android OS already have a back button or back gesture 🤷‍♂️

**Comment by NikunjKhangwal** [comment]
> Yeah, this button is not really necessary. Android OS already have a back button or back gesture 🤷‍♂️

I think adding it was a logical idea, a lot of apps have back button on upper side. It just doesn't translate well in bottom nav.

**Comment by sayyid5416** [comment]
Then maybe adding an option in setting to hide would be great.
